src/test/test_dir.c

```diff
@@ -3014,7 +3016,7 @@ gen_routerstatus_for_umbw(int idx, time_t now)
   if (vrs) {
     vrs->microdesc = tor_malloc_zero(sizeof(vote_microdesc_hash_t));
     tor_asprintf(&vrs->microdesc->microdesc_hash_line,
-                 "m 9,10,11,12,13,14,15,16,17 "
+                 "m 13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28 "
```

Comment:
Should this list be 25-28?
21 is not a supported method, should it be added to the list?
After this branch, 13-24 aren't supported, should they be in the list?

Reply:
8420898a87f58ddfba28da477465b2f0859870b1 will fix this